src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/PanelStatus.tsx

```diff
@@ -11,18 +22,16 @@ export function PanelStatus({ analysisPoints }: { analysisPoints: AnalysisPayloa
 
   if (!isIndexed) {
     status = "Indexing";
-  } else if (!analysisPoints || !executionPoint) {
+  } else if (!analysisPoints) {
     status = "Loading";
   } else if (analysisPoints.error) {
     status = analysisPoints.error === AnalysisError.TooManyPoints ? "10k+ hits" : "Error";
   } else if (analysisPoints.data.length == 0) {
     status = "No hits";
   } else {
-    const points = analysisPoints
-      ? analysisPoints.data.filter(point => BigInt(point.point) <= BigInt(executionPoint))
-      : [];
+    const pointsCount = getPreviousPointsCount(analysisPoints, executionPoint);
 
-    status = `${points.length}/${analysisPoints.data.length}`;
+    status = `${pointsCount}/${analysisPoints.data.length}`;
```

Comment:
The format of "…/5" doesn't seem meaningful or intuitive to me.

Maybe it's because I think of "…" as a _loading_ indicator? This is meant to indicate "not a match".

My first thoughts are that:
* It's useful to keep showing the total count of things (`analysisPoints.data.length`)
* Maybe we could show the closest previous match but dim it out in some way? This could convey "you're roughly here within the total set of these, but you're not exactly on any one of them".

So like, if I selected the 23rd item the UI would look like this:
![Screen Shot 2022-04-22 at 11 08 07 AM](https://user-images.githubusercontent.com/29597/164742510-fccc4821-03c8-4178-a661-8caf77e41053.png)

And then if I played ahead a little– but not far enough to hit the 24th item, it might look like this?
![Screen Shot 2022-04-22 at 11 08 34 AM](https://user-images.githubusercontent.com/29597/164742511-2b06c837-616b-4b80-acb4-844650919b8b.png)

Reply:
Now that I look at it, maybe the ✱ does look a bit better? Not that it's the best, but at least that way it doesn't look like we're loading.

Reply:
> The main problem here is that mainly, during playback, we can't figure out what that numerator is. The executionPoint that lets us order your current pause on/in-between hits isn't available.

That makes sense, that there's no `point` if you're scrubbing, but I (maybe naively) assumed that we could use the current _time_ value to find the nearest point. The only ambiguous cases that come to my mind are:
* If there _multiple_ points at the current time _and_ you aren't actively viewing one, in which case I think it's okay for to just show the last one?
* If you scrubbed _before_ the first point, in which case I think it's okay to show a greyed out "0"?

> Now that I look at it, maybe the ✱ does look a bit better? Not that it's the best, but at least that way it doesn't look like we're loading.

TBH I think the character that most comes to my mind as a "doesn't match" or "unavailable" is `-` but (that's also not a great match here).

Reply:
Agreed that we can approximate it based on time. while scrolling at best we're dealing with pixel level fidelity...